CONTRIBUTING.md

```diff
@@ -28,12 +28,74 @@ We welcome and enourage all kinds of contribution to the project, not only code.
 This includes bug reports, user experience feedback, assistance in reproducing
 issues and more.
 
+## Issues 
+
 This project uses GitHub Issues to track ongoing development, discuss project
 plans, and keep track of bugs.  Be sure to search for existing issues before
 you create another one.
 
 Visit [our Issues page on GitHub to search and submit](https://github.com/eclipse/omr/issues)
 
+### Labelling
+
+Our GitHub issues get labelled by committers in order to allow easier navigation,
+and to flag certain issues as being of interest to certain groups. A subset of the 
+labels are documented below.
+
+* **`beginner`** generally refers to a task that would be suitable for someone new
+  to the project with little experience with either the technology or even
+  open-source projects and processes. They are intended for anyone who wants to
+  gently get their feet wet on building, testing, or familiarizing themselves
+  with part of the code base.
```

Comment:
You might also want to say:
"Multiple labels may be associated with an issue or PR: as many as needed to provide adequate categorization."

Reply:
Done.